includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -121,41 +121,45 @@ public function render_provider_fields() {
 				'input_type'    => 'password',
 				'large'         => true,
 				'class'         => 'classifai-provider-field provider-scope-' . static::ID, // Important to add this.
-				'description'   => sprintf(
-					wp_kses(
-						/* translators: %1$s is the link to register for an IBM Cloud account, %2$s is the link to setup the NLU service */
-						__( 'Don\'t have an IBM Cloud account yet? <a title="Register for an IBM Cloud account" href="%1$s">Register for one</a> and set up a <a href="%2$s">Natural Language Understanding</a> Resource to get your API key.', 'classifai' ),
-						[
-							'a' => [
-								'href'  => [],
-								'title' => [],
-							],
-						]
+				'description'   => $this->feature_instance->is_feature_enabled() ?
```

Comment:
So the `is_feature_enabled` method checks three things:

1. Is the Provider configured properly?
2. Does the current user have access to this Feature?
3. Is the Feature turned on?

This is probably fine when someone is setting things up through the onboarding wizard but if changing things from the settings page, this means the helper text won't show if the Feature isn't turned on. This leads to the scenario where the only way to see the helper text is turning the Feature on and then saving the settings.

I think it may be better to use the `is_configured` method, so we aren't running a user check or enabled check but just checking if the Provider is configured.

Reply:
Hello @dkotter.

Good point. However, instead of just checking if the feature is configured I preferred to check if the feature is configured with the specified provider. The problem was that when using just `is_configured` the feature might have been configured with provider A and that would result in hiding the help texts of provider B for the same service. With my approach, the help texts of a provider are only hidden if the feature is configured with the specified provider.

Please let me know what you think.

Thank you